README.md

```diff
@@ -219,25 +216,6 @@ cannot determine if someone else is already working on it, feel free to ask
 (e.g. by leaving a comment on the associated issue).
```

Comment:
Is this the best way to check if other members of the team are working on a feature for Sawtooth? I personally don't check JIRA comments often (read: at all).